lexik/jwt-authentication-bundle/2.3/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Lexik\\Bundle\\JWTAuthenticationBundle\\LexikJWTAuthenticationBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "etc/": "%ETC_DIR%"
+    },
+    "env": {
+        "JWT_PRIVATE_KEY_PATH": "%kernel.project_dir%/var/jwt/private.pem",
```

Comment:
The difference is that you linked to a YAML file (there it will keep working) but here you are using a Symfony param inside a env var. I don't know if that would work.